hasher-matcher-actioner/hmalib/common.py

```diff
@@ -12,3 +23,132 @@ def get_logger(name=__name__, level=logging.INFO):
     logger = logging.getLogger(name)
     logger.setLevel(level)
     return logger
+
+
+logger = get_logger(__name__)
+s3_client = boto3.client("s3")
+
+THREAT_EXCHANGE_DATA_BUCKET_NAME = os.environ["THREAT_EXCHANGE_DATA_BUCKET_NAME"]
+THREAT_EXCHANGE_DATA_FOLDER = os.environ["THREAT_EXCHANGE_DATA_FOLDER"]
+THREAT_EXCHANGE_PDQ_FILE_EXTENSION = os.environ["THREAT_EXCHANGE_PDQ_FILE_EXTENSION"]
+
+
+@dataclass
+class ThreatExchangeS3Adapter:
```

Comment:
blocking: Yea, this is actually pretty beefy, should be it's own file. 

This also pre-supposes that the file format is csv. May want to mention that in the title.

Reply:
yeah, this is partially my mistake, I recommended putting this in common for now without seeing the full scope of the class. CSV has certain advantages (e.g. s3 has built in parsing) agree that calling it out as a requirement is good and might be something to call out at the python-threatexchange level as well.